vtexdocs/RapiDoc
# Feat/language picker

#### What is the purpose of this pull request?

Add the option to choose the code example language.

#### What problem is this solving?

The only language available was shell.

#### How should this be manually tested?

Open the [preview](https://deploy-preview-225--elated-hoover-5c29bf.netlify.app) and check different api reference pages.

#### Screenshots or example usage

#### Types of changes
